pshtt_csv2mongo.py

```diff
@@ -135,6 +135,9 @@ def store_data(clean_federal, agency_dict, db_config_file):
         # Convert "True"/"False" strings to boolean values (or None)
         boolean_items = (
             'Live',
+            'HTTPS Live',
+            'HTTPS Full Connection',
+            'HTTPS Client Auth Required'
```

Comment:
You are missing a comma at the end of this line

Reply:
I actually checked for missing commas and somehow missed that.  Thanks @dav3r!